stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/state/MessageInputState.kt

```diff
@@ -1,16 +1,23 @@
 package io.getstream.chat.android.common.state
 
 import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.client.models.User
 
 /**
  * Represents the state within the message input.
  *
  * @param inputValue The current text value that's within the input.
  * @param attachments The currently selected attachments.
  * @param action The currently active [MessageAction].
+ * @param validationErrors The list of validation errors.
+ * @param mentionSuggestions The list of users that can be used to autocomplete the mention.
+ * @param cooldownTimer The amount of time left until the user is allowed to send the next message.
  */
 public data class MessageInputState(
     val inputValue: String = "",
     val attachments: List<Attachment> = emptyList(),
     val action: MessageAction? = null,
+    val validationErrors: List<ValidationError> = emptyList(),
+    val mentionSuggestions: List<User> = emptyList(),
```

Comment:
So for now, compose uses `MessageInputState` from `compose` module. This one is from `common` and is used by `ui-components`. I'd prefer to remove `compose` class but it's better to do it later (along with exposing state by `MessageComposerController`) to avoid conflicts hell.